master/reference/calicoctl/resources/node.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Node Resource (node)
+title: Node Resource (Node)
```

Comment:
is Node a reference to the `kind`? If so should it be \`Node\` rather than just Node?

Reply:
I tried that but jekyll doesn't render it correctly for the title section. See screenshot below
![image](https://user-images.githubusercontent.com/5966968/31842535-6e240256-b5a3-11e7-97a7-f618130c16e3.png)